blurring_as_a_service/inference_pipeline/components/detect_and_blur_sensitive_data.py

```diff
@@ -11,20 +12,29 @@
 from mldesigner import Input, Output, command_component
 
 sys.path.append("../../..")
-import yolov5.val as val  # noqa: E402
 
 from blurring_as_a_service.settings.settings import (  # noqa: E402
     BlurringAsAServiceSettings,
 )
+from blurring_as_a_service.settings.settings_helper import (  # noqa: E402
+    setup_azure_logging,
+)
 from blurring_as_a_service.utils.generics import delete_file  # noqa: E402
 
 config_path = os.path.abspath(
     os.path.join(os.path.dirname(__file__), "..", "..", "..", "config.yml")
 )
+
 settings = BlurringAsAServiceSettings.set_from_yaml(config_path)
+log_settings = BlurringAsAServiceSettings.set_from_yaml(config_path)["logging"]
+setup_azure_logging(log_settings, __name__)
+
 aml_experiment_settings = settings["aml_experiment_details"]
 
 
+import yolov5.val as val  # noqa: E402
```

Comment:
Can we add a comment in all these imports that are after the log to explicitly say that they have been moved after the logs so the logging works?
So in the future nobody by mistake reorders them 😆